tests/drivers/pwm/pwm_loopback/boards/nucleo_f103rb.overlay

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pwm/pwm.h>
+
+/ {
+	pwm_loopback_0 {
+		compatible = "test-pwm-loopback";
+		/* first index must be a 32-Bit timer */
+		pwms = <&pwm1 1 0 PWM_POLARITY_NORMAL>,
+			<&pwm3 2 0 PWM_POLARITY_NORMAL>;
+	};
+};
+
+/* 32-Bit timers */
+&timers1 {
+	status = "okay";
+	pwm1: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim1_ch1_pwm_out_pa8>; /* CN8 PIN1 A0 */
+		pinctrl-names = "default";
+	};
+};
+
+&timers3 {
```

Comment:
One comment of the https://github.com/zephyrproject-rtos/zephyr/pull/54019#pullrequestreview-1279336127
was that "All the device nodes should be defined in boards DTS files, and not in samples DTS overlays."
So I expect to find the timers3 node in the board DTS (the pinctrl-0 is taking place in the overlay).